### PR TITLE
Remove redundant scripts and ts-node-dev dependency

### PR DESCRIPTION
**Pull request by haltman-at**
This PR removes redundant scripts from the `db` package; these scripts are no longer needed now that the `truffle db serve` command exists.  It also removes the `ts-node-dev` devDependency, which was only used for these scripts, and was the real reason for this PR, in the hopes that it will help us resolve some dependabot alerts. :)

### CODE REVIEW

File: packages/db/package.json [hunk 1]
```diff
@@ -29,10 +29,6 @@
     "clean": "rm -rf ./dist ./types/schema.d.ts",
     "madge": "madge ./src/index.ts",
     "prepare": "yarn build",
-    "start": "ts-node-dev -r tsconfig-paths/register ./bin/server",
-    "start:debug": "DEBUG=pouchdb:api ts-node-dev -r tsconfig-paths/register ./bin/server",
-    "start:drizzle": "ts-node-dev -r tsconfig-paths/register ./bin/server ./test/truffle-projects/drizzle-box",
-    "start:drizzle:debug": "DEBUG=pouchdb:api ts-node-dev -r tsconfig-paths/register ./bin/server ./test/truffle-projects/drizzle-box",
```
Comment (cds-amal) [inline packages/db/package.json:35]: We can lose the `truffle-projects/` folder now.

**Review by cds-amal — CHANGES_REQUESTED** [review]
Thanks for following up on this @haltman-at. Now there's unused test fixtures we can remove.

File: packages/db/package.json (hunk 1 above)
Comment (haltman-at) [reply packages/db/package.json:35]: Oh, I didn't realize, thanks!  @gnidan didn't mention that so I didn't know offhand whether that was used elsewhere.  (I guess I should likely check myself to be sure...)

Comment (haltman-at) [reply packages/db/package.json:35]: Yup, looks like you're right!  Made that change.

**Review by cds-amal — APPROVED** [review]
Looks good!

**Pull request merged**